specification/streamanalytics/resource-manager/Microsoft.StreamAnalytics/preview/2020-03-01-preview/clusters.json

```diff
@@ -0,0 +1,612 @@
+      "properties": {
+        "value": {
+          "description": "A list of streaming jobs.",
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/ClusterJob"
+          },
+          "readOnly": true
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "The URL to fetch the next set of streaming jobs.",
+          "readOnly": true
+        }
+      }
+    },
+    "ClusterSku": {
+      "description": "The SKU of the cluster. This determines the size/capacity of the cluster. Required on PUT (CreateOrUpdate) requests.",
+      "properties": {
+        "name": {
+          "description": "Specifies the SKU name of the cluster. Required on PUT (CreateOrUpdate) requests.",
+          "type": "string",
+          "enum": [
+            "Default"
+          ],
+          "x-ms-enum": {
+            "name": "ClusterSkuName",
+            "modelAsString": true,
+            "values": [
+              {
+                "value": "Default",
+                "description": "The default SKU."
+              }
+            ]
+          }
+        },
+        "capacity": {
+          "description": "Denotes the number of streaming units the cluster can support. Valid values for this property are multiples of 36 with a minimum value of 36 and maximum value of 216. Required on PUT (CreateOrUpdate) requests.",
+          "type": "integer",
+          "format": "int32",
+          "minimum": 36,
+          "maximum": 216
+        }
+      }
+    },
+    "ClusterProvisioningState": {
+      "description": "The status of the cluster provisioning. The three terminal states are: Succeeded, Failed and Canceled",
+      "readOnly": true,
+      "type": "string",
+      "enum": [
+        "Succeeded",
+        "Failed",
+        "Canceled",
+        "InProgress"
+      ],
+      "x-ms-enum": {
+        "name": "ClusterProvisioningState",
+        "modelAsString": true,
+        "values": [
+          {
+            "value": "Succeeded",
+            "description": "The cluster provisioning succeeded."
+          },
+          {
+            "value": "Failed",
+            "description": "The cluster provisioning failed."
+          },
+          {
+            "value": "Canceled",
+            "description": "The cluster provisioning was canceled."
+          },
+          {
+            "value": "InProgress",
+            "description": "The cluster provisioning was inprogress."
+          }
+        ]
+      }
+    },
+    "JobState": {
+      "description": "The current execution state of the streaming job.",
+      "type": "string",
+      "readOnly": true,
+      "enum": [
+        "Created",
+        "Starting",
+        "Running",
+        "Stopping",
+        "Stopped",
+        "Deleting",
+        "Failed",
+        "Degraded",
+        "Restarting",
+        "Scaling"
+      ],
+      "x-ms-enum": {
+        "name": "JobState",
+        "modelAsString": false,
```

Comment:
I think this should be modelAsString = true to be flexible/backwards compatible unless there is a good reason to be false here

Reply:
fixed